pkg/cwapi/auth.go

```diff
@@ -113,7 +113,7 @@ func (ctx *ApiCtx) Init(cfg string, profile string) error {
 		return err
 	}
 	//start the background go-routine
-	go ctx.pushLoop()
+	go ctx.pushLoop() //nolint:errcheck
```

Comment:
Maybe we should issue guidelines or some kind of rules to be allowed to escape linter ?

Reply:
Maybe add an extra comment on why we bypass linter, would be enough ? You did that below, and it's ok with me.